costs:
  straight: 3
  rotate_left: 6
  rotate_right: 6